system_modules/naprender/src/rendertarget.cpp

```diff
@@ -168,20 +168,21 @@ namespace nap
 
 	void RenderTarget::beginRendering()
 	{
-		glm::ivec2 size = mColorTexture->getSize();
-		std::array<VkClearValue, 2> clearValues = {};
-		clearValues[0].color = { mClearColor[0], mClearColor[1], mClearColor[2], mClearColor[3] };
-		clearValues[1].depthStencil = { 1.0f, 0 };
+		const auto size = mColorTexture->getSize();
+		std::array<VkClearValue, 3> clear_values = {};
+		clear_values[0].color = { mClearColor[0], mClearColor[1], mClearColor[2], mClearColor[3] };
+		clear_values[1].depthStencil = { 1.0f, 0 };
+		clear_values[2].color = { mClearColor[0], mClearColor[1], mClearColor[2], mClearColor[3] };
```

Comment:
Is this the right way to solve this? I understand that the number of clear values must be greater or equal than the number of attachments, but why are there exactly 3 if not more? Can't we properly determine this?